labguide/research/preregistration.md

```diff
@@ -59,3 +60,12 @@ pre-registration should specify:
    - Publications should include
     an explicit “Deviations from pre-registration” section that outlines
     any deviations and their rationale.
+
+### Registered Reports
+
+Whenever possible, we support the submission of Registered Reports.
+
+Registered reports are a publication mechanism in which one submits a manuscript that contains an Introduction and Methods section that describe a study prior to execution of the research; it is essential a written version of a pre-registration.  The manuscript is reviewed on the strength of the rationale and methods, and if deemed sufficient it is given an "in-principle acceptance" such that the journal guarantees to publish the paper regardless of the results, as long as the approved methods were followed.
```

Comment:
essential --> essentially*. 

guarantees --> agrees*. (some journals are POS and may not)

Reply:
New line for every sentence ? 🙏